### PR TITLE
Feature/mir 64/plugin fixes

### PR DESCRIPTION
**Pull request by Daboo1998**

### CODE REVIEW

File: packages/plugin_miro/src/stores/MiroBoardStore.ts/MiroBoardStore.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ const MiroBoardStore = types
         return;
       }
       yield self.api.set('board', data);
```
Comment (PAKovalev) [inline packages/plugin_miro/src/stores/MiroBoardStore.ts/MiroBoardStore.ts:31]: Btw: set function should return value. Let assign response as value.

**Review by PAKovalev — CHANGES_REQUESTED** [review]

**Review by dmitry-yudakov — APPROVED** [review]

File: packages/plugin_miro/src/stores/MiroBoardStore.ts/MiroBoardStore.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ const MiroBoardStore = types
...
-      self.setBoard(data);
+      self.board = data;
```
Comment (PAKovalev) [inline packages/plugin_miro/src/stores/MiroBoardStore.ts/MiroBoardStore.ts:32]: Btw: set function should return value. Let assign response as value.

**Review by PAKovalev — CHANGES_REQUESTED** [review]

**Review by PAKovalev — APPROVED** [review]

File: packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts [hunk 3]
```diff
@@ -37,21 +37,24 @@ const PluginAttributesManager = types
     }),
     set: flow(function* <T>(spaceId: string, key: string, value: T extends undefined ? never : T) {
       if (!self.pluginId) {
-        return;
+        return null;
       }
 
       const repiository = api.spaceAttributeRepository;
```
Comment (aminbksi) [inline packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts:43]: typo `repository`

File: packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts [hunk 4]
```diff
@@ -37,21 +37,24 @@ const PluginAttributesManager = types
...
+        plugin_id: self.pluginId,
+        attribute_name: AttributeNameEnum.STATE,
+        sub_attribute_key: key,
+        value
+      };
+
+      if (value === null) {
+        return yield self.setStateRequest.send(repiository.deleteSpaceSubAttribute, body);
+      } else {
+        return (yield self.setStateRequest.send(repiository.setSpaceSubAttribute, body))[key];
```
Comment (aminbksi) [inline packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts:56]: wouldn't better to first map the response and then return the value?

**Pull request merged**